test/index.test.js

```diff
@@ -55,64 +55,70 @@ const parseScript = (() => {
 })();
 
 describe('fixtures', () => {
+	const dirs = [];
 	fs.readdirSync(FIXTURES_DIR).forEach(fixtureDir => {
```

Comment:
this could probably be a map + filter for a more functional approach

Reply:
👍  make sense!